scripts/build-appbuilder.sh

```diff
@@ -1,11 +1,34 @@
 #!/bin/bash
+SENTRY_CLI="node_modules/.bin/sentry-cli"
+SENTRY_ORG="shapediver"
+SENTRY_PROJECT="app-builder"
+
+# trap exceptions and restore sentryconfig.ts
+trap 'if [ -f "sentryconfig.ts.bak" ]; then mv sentryconfig.ts.bak sentryconfig.ts; fi' EXIT
 
 version=$1
 if [ -z "$version" ]; then
     echo "Usage: $0 <version>"
     exit 1
 fi
 
+# should we deploy, or just build?
+deploy=$2
+
+# Check if sentry-cli exists
+if [ ! -z "$deploy" -a ! -x "$SENTRY_CLI" ]; then
+    echo "Could not find sentry-cli."
+    exit 1
+fi
+
+# Check if sentryconfig.local.ts exists, copy it to sentryconfig.ts if it does
+build_timestamp=$(date +'%Y-%m-%d_%H:%M')
+if [ -f "sentryconfig.local.ts" -a ! -z "${deploy}" ]; then
+    echo "Configuring sentry for build timestamp: ${build_timestamp}"
+    mv sentryconfig.ts sentryconfig.ts.bak
+    sed -e "s/BUILD_TIMESTAMP/${build_timestamp}/" sentryconfig.local.ts > sentryconfig.ts
+fi
```

Comment:
@LemToUp @MajorMeerkatThe3rd the deployment scripts overwrite `sentryconfig.ts` in order to activate sentry for deployed code